app/scripts/modules/plugins/tsconfig.json

```diff
@@ -0,0 +1,38 @@
+{
+  "buildOnSave": false,
+  "compileOnSave": true,
+  "compilerOptions": {
+    "allowJs": false,
+    "baseUrl": "./src",
+    "declaration": true,
+    "declarationDir": "lib",
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
+    "jsx": "react",
+    "lib": ["es2016", "dom"],
+    "moduleResolution": "node",
+    "module": "esnext",
+    "noEmitHelpers": false,
+    "noImplicitAny": true,
+    "noImplicitReturns": true,
+    "noImplicitThis": false, // should really get to a place where we can turn this on
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "outDir": "lib",
+    "pretty": true,
+    "removeComments": false,
+    "rootDir": "./src",
+    "skipLibCheck": true,
+    "sourceMap": true,
+    "inlineSources": true,
+    "strictNullChecks": false, // should really get to a place where we can turn this on
+    "target": "es6",
+    "typeRoots": ["../../../../node_modules/@types"],
+    "paths": {
+      "@spinnaker/core": ["../../core/lib"],
+      "core/*": ["core/src/*"]
```

Comment:
I don't know much/anything about TS configs but `core/*` is defined as `../../core/lib/*` in the `kubernetes`  and `aws` modules. Not sure if defining it as `../../core/src/*` will cause any issues 🤷‍♂ .

Reply:
I don't know. I copied it from the `docker` module